README.md

```diff
@@ -38,11 +44,12 @@ This action can be used to redeploy application on the DigitalOcean's [App Platf
                       ]'
```

Comment:
Add some example data

Reply:
I intentionally skipped the example data so that user can copy the format and paste it in their workflow. Because while doing research for headers i found that while reading documentation many users prefer to copy from documentation prompt.